monkey/infection_monkey/exploit/sshexec.py

```diff
@@ -233,3 +214,13 @@ def _exploit_host(self) -> ExploiterResultData:
 
             logger.error(self.exploit_result.error_message)
             return self.exploit_result
+
+    def _make_agent_executable(self, ftp: paramiko.sftp_client.SFTPClient):
```

Comment:
Lets leave a comment that this can be a part of `HostExploiter` as we go further.